uint8 AREA_NONE=0
uint8 AREA_NAVIGATION=1
uint8 AREA_MOWING=2
uint8 AREA_PROHIBITED=3

string name
uint8 area_type
geometry_msgs/Polygon area